playground/infrastructure/datastore_client.py

```diff
@@ -83,54 +83,53 @@ def save_to_cloud_datastore(self, examples_from_rep: List[Example], sdk: Sdk, or
         examples_ids_before_updating = self._get_all_examples(sdk, origin)
 
         # loop through every example to save them to the Cloud Datastore
```

Comment:
below is a long-lasting issue with updating big number of entities in a transaction
really complicates testing